x-pack/legacy/plugins/painless_lab/public/components/output_pane/output_tab.tsx

```diff
@@ -6,21 +6,20 @@
 
 import React from 'react';
 import { EuiCodeBlock, EuiSpacer } from '@elastic/eui';
-import { i18n } from '@kbn/i18n';
 
 import { formatResponse } from '../../lib/helpers';
 import { Response } from '../../common/types';
 
 interface Props {
-  response?: Response;
+  response: Response | null;
 }
 
-export function OutputTab({ response = {} }: Props) {
+export function OutputTab({ response }: Props) {
   return (
     <>
       <EuiSpacer size="m" />
       <EuiCodeBlock language="json" paddingSize="s" isCopyable>
-        {formatResponse(response.success || response.error)}
+        {formatResponse(response ?? undefined)}
```

Comment:
Nit: if we initialize `response` as `undefined` in `useSubmitCode`, then would we need `?? undefined` here or need to change the Props interface? I have a hard time recalling an instance where I got value out of a `null` value and so I've generally been defaulting to things being `undefined` unless there is a really good reason to use `null`. So far it's seemed to help simplify code in my experience.